easybuild/easyconfigs/r/Ruby/Ruby-3.0.1-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,164 @@
+name = 'Ruby'
+version = '3.0.1'
+
+homepage = 'https://www.ruby-lang.org'
+description = """Ruby is a dynamic, open source programming language with
+ a focus on simplicity and productivity. It has an elegant syntax that is
+ natural to read and easy to write."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+source_urls = ['https://cache.ruby-lang.org/pub/ruby/%(version_major_minor)s']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['369825db2199f6aeef16b408df6a04ebaddb664fb9af0ec8c686b0ce7ab77727']
+
+builddependencies = [('binutils', '2.37')]
+
+exts_default_options = {
+    'source_urls': ['https://rubygems.org/downloads/'],
+    'source_tmpl': '%(name)s-%(version)s.gem',
+}
+
+# !! order of packages is important !!
+# some packages have dependencies with minimum and maximum version requirements
+# each version is picked as high as possible to fullfill all requirements
+# packages updated on 2021-07-02
+exts_list = [
```

Comment:
@ItIsI-Orient all these extensions should ideally be updated to their most recent version, if possible.

Reply:
I realize this is a lot of work, maybe you can ask in Slack or the mailing list if someone has a script for that?